src/PHPDebt/Console/Command/CodeSnifferConfigCommand.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace PHPDebt\Console\Command;
+
+use PHP_CodeSniffer\Config;
+use PHP_CodeSniffer\Runner;
+use PHPDebt\PHPDebtMD;
+use SebastianBergmann\FinderFacade\FinderFacade;
+use SebastianBergmann\PHPLOC\Analyser;
+use Symfony\Component\Console\{
+	Command\Command,
+	Input\InputArgument,
+	Input\InputInterface,
+	Input\InputOption,
+	Output\OutputInterface
+};
+
+/**
+ * The class for all `phpdebt` command.
+ *
+ * @package PHPDebt\Console\Command
+ */
+class CodeSnifferConfigCommand extends Command {
```

Comment:
The description says it's the whole `phpdebt` command, but the class is named like it's for configuring code sniffer.

Reply:
Thanks for catching it, I've written it initially as was thinking that I would be dealing with here code related only to the PHPCS, later my scope changed, but forgot to rename the class. This is fixed now.